### PR TITLE
Update DisplayList tests to explicitly select the Roboto font

### PR DESCRIPTION
**Pull request by jason-simmons**
Skia is removing the API for constructing a default typeface (see https://issues.skia.org/issues/305780908)

Fixes https://github.com/flutter/flutter/issues/137565

### CODE REVIEW

**Comment by jason-simmons** [comment]
(Requires https://github.com/flutter/engine/pull/47491)

@kjlubick

File: display_list/BUILD.gn [hunk 1]
```diff
@@ -201,10 +202,6 @@ if (enable_unittests) {
   }
 }
 
-fixtures_location("display_list_benchmarks_fixtures") {
```
Comment (flar) [inline display_list/BUILD.gn:204]: Is this really unused?

**Review by flar — APPROVED** [review]
LGTM - other than my question about the previous benchmark fixtures...

File: display_list/BUILD.gn (hunk 1 above)
Comment (jason-simmons) [reply display_list/BUILD.gn:204]: I did not see any references to fixtures in the DL benchmarks.  The benchmarks appear to be running as expected with this change.

**Comment by kjlubick** [comment]
LGTM once the compilation issue is sorted out

**Pull request merged**